cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,40 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
+            nargs='+',
+            help='The slug for the page you wish to delete'
+        )
+        parser.add_argument(
+            '--id',
+            nargs='+',
+            help='The ID for the page you wish to delete'
+        )
+
+    def handle(self, *args, **options):
+        page_to_delete = None
+        if options['slug']:
+            slug = options['slug'][0]
```

Comment:
Instead of using `nargs='+'` and then referring to e.g. `options['slug'][0]` can you just use `nargs=?` to make the arguments optional?